idaes/tests/test_ipopt.py

```diff
@@ -36,6 +33,7 @@ def test_ipopt_idaes_config():
     """
     Test that the default solver options are set
     """
+    isolve.use_idaes_solver_configuration_deafults()
```

Comment:
Typo in method name: defaults

This reminds me that someone pointed out the "constraints" was misspelled somewhere ("constaints"?) in the scaling tools too (suffix name or storage dict?).